lib/active_record/connection_adapters/percona_adapter.rb

```diff
@@ -74,6 +74,12 @@ def initialize(connection, _logger, connection_options, _config)
         @prepared_statements = false
       end
 
+      def write_query?(sql) # :nodoc:
```

Comment:
Needs to be [implemented](https://github.com/rails/rails/blob/6-1-stable/activerecord/lib/active_record/connection_adapters/abstract/database_statements.rb#L112). Brought in from [mysql](https://github.com/rails/rails/blob/6-1-stable/activerecord/lib/active_record/connection_adapters/mysql/database_statements.rb#L22-L29).